datahub-web-react/src/app/shared/tags/TagTermGroup.tsx

```diff
@@ -198,7 +216,10 @@ export default function TagTermGroup({
             {/* uneditable tags are provided by ingestion pipelines exclusively */}
             {uneditableTags?.tags?.map((tag) => {
                 renderedTags += 1;
-                if (maxShow && renderedTags > maxShow) return null;
+                if (maxShow && renderedTags > maxShow)
+                    return (
+                        <TagText>{uneditableTags?.tags ? `+${uneditableTags?.tags?.length - maxShow}` : null}</TagText>
+                    );
```

Comment:
same idea here! if there are more than 1 `uneditableTags` than the maxShow (ie. maxShow = 3 and `uneditableTags.length` = 5) then we will render "+2" multiple times

Reply:
required changes are added.